web/src/api/media.ts

```diff
@@ -81,12 +81,19 @@ function getFallbackCharacter(from: unknown, idx: number): string {
 export const getAvatarURL = (userID: UserID, content?: UserProfile | null): string | undefined => {
 	const fallbackCharacter = getFallbackCharacter(content?.displayname, 0) || getFallbackCharacter(userID, 1)
 	const backgroundColor = getUserColor(userID)
-	const [server, mediaID] = parseMXC(content?.avatar_url)
+	let server: string | undefined
+	let mediaID: string | undefined
+	if (content?.avatar_file) {
+		[server, mediaID] = parseMXC(content.avatar_file.url)
+	} else {
+		[server, mediaID] = parseMXC(content?.avatar_url)
+	}
 	if (!mediaID) {
 		return makeFallbackAvatar(backgroundColor, fallbackCharacter)
 	}
+	const encrypted = !!content?.avatar_file
```

Comment:
This looks like it should be `!!`?

Reply:
It is?

Reply:
It's encrypted if `avatar_file` is present, a single `!` will return true if avatar_file is not present

Reply:
Oh there's some weird font thing on github that's hiding the first one 🧐